### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by TomSnijders**
# Description

This is effectively the submission to CRAN for version 1.3.0.
Nothing new was added, except that the test using testthat was dropped.
This was accepted by CRAN.

### CODE REVIEW

File: DESCRIPTION [hunk 1]
```diff
@@ -2,8 +2,8 @@ Encoding: UTF-8
 Package: RSiena
 Type: Package
 Title: Siena - Simulation Investigation for Empirical Network Analysis
-Version: 1.2.34
-Date: 2021-04-30
+Version: 1.3.0
+Date: 2021-05-02
```
Comment (jhollway) [inline DESCRIPTION:6]: 2021-05-12?

File: docs/manual/RSiena_Manual.tex [hunk 2]
```diff
@@ -16002,11 +16002,25 @@ \section{Changes compared to earlier versions}
 \begin{small}
 \begin{itemize} % Also update version number in earlier places.
 
+\item 2021-05-02 CRAN/GitHub, package version 1.3.0.
+\medskip
+
+The cumulative importance of all changes in the past two years
+(including Generalized Method of Moments and continuous
+behavioral variables) justifies that the version number
+now proceeds from 1.2.x to 1.3.0.
```
Comment (jhollway) [inline docs/manual/RSiena_Manual.tex:16011]: Maybe add this note to the NEWS file?

File: docs/manual/SubmittingToCRAN.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
+# Submitting the package to CRAN.   
+  
+This text contains the main things to be noted and remembered.   
+
```
Comment (jhollway) [inline docs/manual/SubmittingToCRAN.md:4]: I actually use `devtools::submit_cran()`

File: docs/manual/SubmittingToCRAN.md [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+   my local Github directory, I got warning messages   
+   "Found the following Makefile(s) with CR or CRLF line endings: src/Makevars   
+    Some Unix 'make' programs require LF line endings."    
+   These were not obtained any more when I had copied files from 
+   https://github.com/snlab-nl/rsiena/archive/refs/tags/v1.2.34.zip
+   and made a tarball from those (after making some additional changes). 
+   Apparently, there is something with line endings that is not correct
+   on my own Windows setup for GitHub.
+3. When all checks passed well (Windows, Mac, Unix), the reverse dependencies
+   should be checked.
```
Comment (jhollway) [inline docs/manual/SubmittingToCRAN.md:37]: https://github.com/r-lib/revdepcheck

File: docs/manual/SubmittingToCRAN.md [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+
+-  [RSiena website at CRAN](https://cran.r-project.org/web/packages/RSiena/)    
+There will be questions about how any issues arising here were addressed.   
+In particular,
+[RSiena check results](https://CRAN.R-project.org/web/checks/check_results_RSiena.html)
+
+## What you need on your machine (in addition to the package itself)
+- R-release and R-devel should be installed, with all dependencies. 
+- For the last step, also the reverse dependencies (not many),
+  with all their dependencies, should be installed for R-devel.
```
Comment (jhollway) [inline docs/manual/SubmittingToCRAN.md:22]: I'm using `rhub::check_for_cran()`

**Comment by TomSnijders** [comment]
CRAN version 1.3.0

**Pull request merged**